index.html

```diff
@@ -72,6 +75,74 @@ <h2>
 
                 <!-- Please keep the team pages in alphabetical order, too. -->
 
+                <!-- Best Practices Team -->
+                <section data-background-image="images/best-practices-team.webp" data-background-color="#b6ccd3">
+                    <h2>
+                        Best Practices Team
+                    </h2>
+                    <ul>
+                        <li>Daniel Siepmann</li>
+                        <li>Oliver Klee</li>
+                        <li>Łukasz Uznański</li>
+                    </ul>
+                </section>
+                <section data-background-color="#b6ccd3">
+                    <h2>
+                        Best Practices Team
+                    </h2>
+                    <h3>
+                        Our purpose ("Why?")
+                    </h3>
+                    <p>
+                        We help TYPO3 developers build high-quality extensions and sites.
+                    </p>
+                </section>
+                <section data-background-color="#b6ccd3">
+                    <h2>
+                        Best Practices Team
+                    </h2>
+                    <h3>
+                        Our mission ("What?")
+                    </h3>
+                    <p>
```

Comment:
Currently, with the (by default) centered text, this is a bit hard to read. Maybe we can use bullet points instead?

Reply:
That makes sense, I've updated the markup.